index.js

```diff
@@ -381,6 +406,14 @@ app.post('/api/door', Validator.body(['coreid', 'data']).exists(), async (reques
       response.status(200).json(errorMessage)
     }
   } catch (err) {
+    try {
+      await db.rollbackTransaction(client)
+      helpers.logError(`handleAlert: Rolled back transaction because of error: ${err}`)
+    } catch (error) {
+      // Do nothing
+      helpers.logError(`handleAlert: Error rolling back transaction: ${error} Rollback attempted because of error: ${error}`)
```

Comment:
I think the `handleAlert` part in these log messages is leftover from a copy-and-paste. It should probably be changed to something representative on this function, `handleDoorEvent` maybe?

Reply:
oh woops, yeah. Actually this points to a difference in organization where here we're doing everything in the route itself instead of just calling a function to do the actual work, and only doing validation in the router